Code/GraphMol/Descriptors/WHIM.cpp

```diff
@@ -172,7 +172,8 @@ std::vector<double> getWhimD(std::vector<double> weightvector,
         }
         if (std::fabs(Scores(j, i) + Scores(k, i)) <= th) {
           // those that are close opposite & not close to the axis!
-          ns += 1;  // check only once the symmetric none null we need to add +2!
+          ns +=
```

Comment:
odd formatting

Reply:
yeah, that was clang-format. I just moved the comments around and that cleared up